### ISSUE TITLE
Improve field constructor documentation

### ISSUE THREAD

**Issue by jroper** [playframework/playframework#5172]
The [custom field constructor documentation](https://www.playframework.com/documentation/2.4.x/ScalaCustomFieldConstructors) neglects to say where you should (or can) put the field constructor template.  The answer is of course anywhere, but I think it would be good if the documentation suggested a filename relative to the project directory, both for the field constructor and the `MyHelpers` object, and a little explanation around choosing the location of the file and what would need to change if you used a different location would be good.

**Comment by CapeSepias** [comment]
I agree. Currently its very time-consuming as an non-seasoned Play/scala user to make this simple magic work.... Would love to see this improvement written by a seasoned person.

**Comment by RommelTJ** [comment]
I'm taking a look at this.

**Comment by RommelTJ** [comment]
@jroper @CapeSepias This is very old, but I took a look at this and I agree. The existing documentation was a bit confusing. 

I made an attempt on updating it here: https://github.com/playframework/playframework/pull/12892
